src/metatrain/experimental/phace/modules/message_passing.py

```diff
@@ -0,0 +1,311 @@
+import copy
+from typing import Dict, List, Tuple
+
+import metatensor.torch
+import torch
+from metatensor.torch import Labels, TensorBlock, TensorMap
+
+from .cg import cg_combine_l1l2L
+from .layers import Linear
+from .radial_basis import RadialBasis
+from .tensor_sum import EquivariantTensorAdd
+
+
+class DummyAdder(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(
+        self, tmap_1: metatensor.torch.TensorMap, tmap_2: metatensor.torch.TensorMap
+    ) -> metatensor.torch.TensorMap:
+        return metatensor.torch.TensorMap(
```

Comment:
```suggestion
        self, tmap_1: TensorMap, tmap_2: TensorMap) -> TensorMap:
        return TensorMap(
```

you have the names in scope